test/e2e/specs/editor/blocks/list.spec.js

```diff
@@ -1231,4 +1231,38 @@ test.describe( 'List', () => {
 
 		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
 	} );
+
+	test( 'selects all transformed output', async ( { editor, page } ) => {
+		await editor.insertBlock( {
+			name: 'core/list',
+			innerBlocks: [
+				{ name: 'core/list-item', attributes: { content: '1' } },
+				{ name: 'core/list-item', attributes: { content: '2' } },
+			],
+		} );
+
+		await editor.selectBlocks(
+			page.locator( 'role=document[name="Block: List"i]' )
+		);
+
+		await page
+			.getByRole( 'button', {
+				name: 'List',
+				description: 'List: Change block type or style',
+			} )
+			.click();
```

Comment:
I think the `description` field is only needed because we're not limiting the results. Selectors are strict by default, which means if a selector matches multiple elements then it will fail. We can chain selectors to filter out those that are not what we're targeting. 

In this case, I think we can do this:

```suggestion
		await page
			.getByRole( 'region', { name: 'Block tools' } )
			.getByRole( 'button', { name: 'List' } )
			.click();
```

Reply:
It's not that the description was needed, it's that I'd like to use it over more specificity. I would have liked to simply use `page.getByRole( 'button', { description: 'List: Change block type or style' } )`, but it did not work ("Error: strict mode violation: "role=button" resolved to 20 elements"). Do you know why? I though the description should limit the results to 1.

Reply:
I see!

I think the `description` field is never a valid option in [`getByRole`](https://playwright.dev/docs/api/class-page#page-get-by-role)?

Reply:
Ah! I was looking at the [wrong API docs](https://testing-library.com/docs/queries/byrole/). Strange that it's not an option here, it would be useful.

Reply:
I still think we should chain selectors here though. Selecting `getByRole('button', { name: 'List' })` is not immediately readable for which element exactly is being selected. The test will break if there's a random button with the same name added to the page. To increase the specify level, we should make it clear that the button is from the `Block tools` region as suggested above. WDYT?